### ISSUE TITLE
URL rewriting

### ISSUE THREAD

**Issue by kberzinch** [kberzinch/cs1331.org#4]
The current method for accessing different URLs is janky. Set up proper server-side rewriting to do this transparently.

**Comment by guyfleeman** [comment]
Is this done via Nginx. Do you have an example of what you mean?

**Comment by kberzinch** [comment]
Yes it's in Nginx, no I don't have an example rn. Will post the configuration when I set it up.

**Comment by guyfleeman** [comment]
Does this mean most of the PHP goes away? That would be really clean.

**Comment by kberzinch** [comment]
Short version no. It just makes the URL look better.

**Comment by kberzinch** [comment]
Okay, here's what needs to happen with this:
 - The server has been updated to just load `index.php` if everything else fails. You can get the URI from `$_SERVER['REQUEST_URI']`.
 - Use that URI to pull the content from the proxy.
 - You no longer need to modify relative hrefs in the HTML, but if there are absolute URLs you will still want to rewrite those.

**Comment by guyfleeman** [comment]
So the relative mod goes away and instead if the GET var we use SERVER?

**Comment by kberzinch** [comment]
Basically. You have the GET parameter giving you the whole URL though; the SERVER parameter only gives you the path (the part after the domain.)
